executor/executor_linux.h

```diff
@@ -182,9 +182,11 @@ static bool use_cover_edges(uint32 pc)
 
 static bool use_cover_edges(uint64 pc)
 {
-#if defined(__i386__) || defined(__x86_64__)
+#if GOARCH_amd64 || GOARCH_arm64
```

Comment:
Do we actually need this ifdef at all? Maybe just check for `is_gvisor`?

Reply:
This check is rather expensive, it's a load for every PC we ever get in coverage.
With both amd64 and arm64 here the benefit becomes moot since it's main arches we care about. But there are still arm/riscv that we run in qemu emulation, for these it would be good to not do this check.